docs/pages/setup/guides/terraform-provider/reference.mdx

```diff
@@ -0,0 +1,422 @@
+|        `impersonate.where`|      string | Optional advanced matcher                                       |
+|                    `rules`|         set | List of rules and their access levels. Rules are a high level construct used for access control |
+|            `rules.actions`| string list | Actions specifies optional actions taken when this rule matches |
+|          `rules.resources`| string list | List of resources                                               |
+|              `rules.verbs`| string list | List of verbs                                                   |
+|              `rules.where`|      string | Additional advanced matcher                                     |
+
+#### `spec.allow.request` and `spec.deny.request`
+
+`request` represents matcher for allow/deny restrictions on access-requests.
+
+| Name                      | Type        | Description                                                     |
+|---------------------------|-------------|-----------------------------------------------------------------|
+|                    `roles`| string list | Name of roles which will match the request rule                 |
+|      `suggested_reviewers`| string list | List of reviewer suggestions. These can be teleport usernames, but that is not a requirement. |
+|              `annotations`|         set | Annotations                                                     |
+|          `annotations.key`|      string | Annotation name                                                 |
+|        `annotations.value`| string list | Annotation values                                               |
+|          `claims_to_roles`|         set | Specifies a mapping from claims (traits) to teleport roles      |
+|    `claims_to_roles.claim`|      string | Claim name                                                      |
+|    `claims_to_roles.roles`| string list | List of static Teleport roles to match                          |
+|    `claims_to_roles.value`|      string | Claim value to match                                            |
+|               `thresholds`|         set | List of thresholds, one of which must be met in order for reviews to trigger a state-transition. If no thresholds are provided, a default threshold of 1 for approval and denial is used. |
+|          `thresholds.name`|      string | Threshold name                                                  |
+|       `thresholds.approve`|      number | Number of matching approvals needed for state-transition        |
+|          `thresholds.deny`|      number | Number of denials needed for state-transition                   |
+|        `thresholds.filter`|      string | Optional predicate used to determine which reviews count toward this threshold. |
+
+#### `spec.allow.review_requests` and `spec.deny.review_requests`
+
+`review_requests` is a matcher for allow/deny restrictions on  access reviews.
```

Comment:
Extra space. Also, `allow/deny` implies that these are interchangeable or the same. Can we say `allow`, `deny` instead?